src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.FieldDelegatingCodeAction.cs

```diff
@@ -42,11 +43,9 @@ protected override async Task<Document> GetChangedDocumentAsync(CancellationToke
                 //
                 // Otherwise, just generate a normal constructor that assigns any provided
                 // parameters into fields.
-                var parameterToExistingFieldMap = new Dictionary<string, ISymbol>();
+                var parameterToExistingFieldMap = ImmutableDictionary.CreateBuilder<string, ISymbol>();
```

Comment:
switched to immutable types across our helpers/features for several pieces of data we pass around.